Automattic/wp-calypso
# State: Remove site receive effects on site updates

Unfortunately, `/me/sites` breaks the WP version in `.updates` for Jetpack sites - it rounds `4.7.3` to `4`. There might be other inconsistencies too, so this PR removes the receive/update effects of `/me/sites` on the sites updates reducer. We can always get it back if we wish.

This will allow us to keep the site updates endpoint as the single source of truth.

To test: 
* Checkout this branch
* Verify all site updates reducer tests pass:

```
npm run test-client client/state/sites/updates/test/reducer.js
```
